File: C/Functions/Call by value/f_1.c
/*
C Program to Check Prime Number By Creating a Function 
*/

#include <stdio.h>

int check_prime(int num);//function declaration

int main(){
    int num = 0;
    printf("Enter number to check its prime or not!: ");
    scanf("%d",&num);

    int y = check_prime(num);// function call---> num is arguments
    if(y==1){
        printf("Its not a prime number!");
    }
    else{
        printf("Its a prime number!");
    }
    return 0;
}
int check_prime(int temp) {//function definition
    if(temp ==0||temp ==1){
        return 1;
    }
    for(int i=2; i<=temp/2; i++){
        if ((int)temp%i ==0) {
            return 1;
        }
    }
    return 0;
}

File: College/Second Year/OOPs/Sem-1_Java/Exp02/F1.java
public class F1 {
    public static void main(String[] args) {
        int[] numbers = new int[100];
        
        for (int num = 2; num <= 100; num++) {
            numbers[num - 1] = num;
        }
        
        for (int place = 2; place <= 100; place++) {
            if (numbers[place - 1] != 0) {

            	for (int x = place * 2; x <= 100; x += place) {
                    numbers[x - 1] = 0;
                }
            }
        }
        
        System.out.println("Prime numbers");
        for (int num = 1; num <= 100; num++) {
            if (numbers[num - 1] != 0 && num != 1) {
                System.out.print(num + " ");
            }
        }
    }
}
